How to read review by phases (#1109)
italia/eidas-it-wallet-docs: docs/it/how-to-read-spec.rst

Reviewer: ```suggestion

```
Concetto già espresso nella intro della sezione

  Fornisce dettagli tecnici supplementari, modelli di implementazione e framework di test, inclusa la gestione delle istanze di applicazioni mobili, specifiche di integrazione della piattaforma nazionale e matrici di test complete per la validazione dell'ecosistema.

Percorsi di lettura per ruolo

> ^^^^^^^^^^^^^^^^^^^^^^

Reviewer 2: ```suggestion
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

  Fornisce dettagli tecnici supplementari, modelli di implementazione e framework di test, inclusa la gestione delle istanze di applicazioni mobili, specifiche di integrazione della piattaforma nazionale e matrici di test complete per la validazione dell'ecosistema.

Lettura per ruolo

> ^^^^^^^^^^^^^^^^^

Reviewer: ```suggestion
Percorsi di lettura per ruolo
^^^^^^^^^^^^^^^^^^^^^^
```

Erano rimasti tre titoli

> Lettura rapida per obiettivi  

Reviewer: ```suggestion

```

Superato

""""""""""""""""""""""""""""

> Percorsi di lettura per ruolo

Reviewer: ```suggestion

```

Superato

> **Relying Party** — *Verificare le credenziali presentate:* :ref:`relying-party-solution:Soluzione di Relying Party`, :ref:`credential-presentation:Presentazione dell'Attestato Elettronico`, :ref:`relying-party-endpoints:Endpoint della Relying Party`; *formati e validità:* :ref:`digital-credential-management:Gestione degli Attestati Elettronici`; *sicurezza:* :ref:`security-privacy-considerations:Considerazioni di Sicurezza e Privacy`.

Reviewer: ```suggestion

```

Modalità di lettura superata dalla lettura successiva per ruoli e fasi di progetto

Lettura per fasi di progetto  
""""""""""""""""""""""""""""

> Per un accesso semplificato agli argomenti di queste specifiche, di seguito viene presentato un percorso di lettura per ruoli che tiene conto della fase in cui si trovano a operare all’interno del sistema IT-Wallet con un rimando preciso alla sezione di interesse del documento. 

Reviewer: ```suggestion
Percorsi di lettura per ruolo
""""""""""""""""""""""""""""

Per facilitare la consultazione degli argomenti trattati in queste specifiche, di seguito sono riportati percorsi di lettura organizzati per ruolo. Ogni percorso è strutturato in base alle diverse fasi di adesione e partecipazione al Sistema IT-Wallet e contiene riferimenti diretti alle sezioni più rilevanti per ciascun ruolo. 

I percorsi di lettura proposti hanno finalità orientativa e non sostituiscono la necessità di consultare le altre sezioni quando è richiesta una comprensione più ampia del sistema. 

Per le entità interessate a operare in più ruoli, si raccomanda di approfondire tutti i percorsi di lettura relativi ai ruoli di interesse. 
```

Fonti Autentiche (Authentic Source)  
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

L’Authentic Source è il proprietario del dato. Il suo ruolo è garantire che le informazioni originali siano trasmesse correttamente e rese disponibili ai sistemi di emissione in modo sicuro e sempre aggiornato.  

> **Focus**: disponibilità del dato, correttezza delle informazioni trasmesse, allineamento costante tra il database originale e le credenziali emesse.  

Reviewer: ```suggestion
Fonte Autentica
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

La Fonte Autentica si occupa di rendere disponibili i dati affidabili alla base dei (Q)EAA e di garantirne e gestirne in modo sicuro l’assoluta accuratezza e integrità. 
```

> **Fase 1: Scoperta del sistema IT-Wallet**

Reviewer: ```suggestion
**Fase 1: Scoperta**
```

> **Obiettivo**: conoscere il contesto di riferimento, i requisiti tecnici e la normativa per entrare a far parte della Trust Chain (catena di fiducia) su cui si fonda tutto il sistema.  

Reviewer: ```suggestion
Comprendere il funzionamento generale dell’ecosistema, l’architettura tecnica e l’Infrastruttura di Trust. 
```

- **Sezione** :ref:`introduction:Introduzione`: per comprendere l’ambito e il linguaggio normativo dell'ecosistema IT-Wallet. 

- **Sezione** :ref:`architecture-overview:Panoramica dell'Architettura`: per conoscere l’Architettura del sistema IT-Wallet, in termini di governance e processi operativi abilitati. 

- **Sezione** :ref:`trust-infrastructure:L'Infrastruttura di Trust`: per conoscere i requisiti chiave della Trust Chain e come essere registrati nella Trust List per poter operare sia a livello nazionale che europeo.  

> - **Sezione** :ref:`defined-terms-and-references:Termini Definiti e Riferimenti`: per conoscere i riferimenti normativi, i termini definiti e gli standard tecnici per consentire un'interoperabilità sicura e corretta tra tutti i partecipanti (puoi ricorrere a questa sezione per ogni dubbio su terminologia, standard di riferimento, acronimi...) 

Reviewer: ```suggestion
- **Sezione** :ref:`introduction:Introduzione`: Scopo e contesto normativo del Sistema IT-Wallet.

- **Sezione** :ref:`architecture-overview:Panoramica dell'Architettura`: Panoramica dell’architettura del Sistema IT-Wallet in termini di governance e processi operativi abilitati. 

- **Sezione** :ref:`trust-infrastructure:L'Infrastruttura di Trust`: Requisiti chiave del modello di fiducia federato e dei meccanismi di valutazione della fiducia tra le entità.

- **Sezione** :ref:`defined-terms-and-references:Termini Definiti e Riferimenti`: Terminologia completa, riferimenti normativi, documentazione, risorse e strumenti aggiuntivi, linee guida per i contributi. 
```

**Fase 2: Design**

> **Obiettivo**: conoscere i requisiti di implementazione degli attributi ritenuti necessari per tradurre i dati in un formato digitale standardizzato (EAA) riconosciuto dal Wallet.  

Reviewer: ```suggestion
**Fase 2: Progettazione**

Comprendere i requisiti e progettare gli aspetti, le funzionalità e le caratteristiche specifiche del (Q)EAA da emettere. 
```

> - **Sezione** :ref:`functionalities:Design dell'Esperienza Utente`: per conoscere le modalità di ottenimento degli Attestati Elettronici da parte dell’utente (riferimento al paragrafo :ref:`functionalities:Ottenimento degli Attestati Elettronici di Attributi`). 

Reviewer: ```suggestion
- **Sezione** :ref:`functionalities:Design dell'Esperienza Utente`: Requisiti chiave relativi alle modalità con cui consentire agli Utenti di ottenere i (Q)EAA, alla struttura dei (Q)EAA, al loro stato e alla loro gestione nel tempo.
```

> - **Sezione** :ref:`entities:Entità`: per conoscere i requisiti di implementazione e in particolare gli attributi necessari per fornire l'Attestato Elettronico richiesto dall'Utente (riferimento a paragrafo :ref:`authentic-sources:Fonti Autentiche`) 

Reviewer: ```suggestion

```
Spostato in implementazione

> **Sezione** :ref:`digital-credential-management:Gestione degli Attestati Elettronici`:  per conoscere come i dati autorevoli e gli attributi diventano Credenziali Elettroniche e in cosa consiste il loro ciclo di vita.

Reviewer: ```suggestion
**Sezione** :ref:`digital-credential-management:Gestione degli Attestati Elettronici`: Requisiti tecnici e funzionali relativi al ciclo di vita dei (Q)EAA. 

- **Sezione** :ref:`e-service-pdnd-template:PDND e-Service Template`: Modello di riferimento contenente tutti i metadati tecnici e descrittivi necessari per la definizione dell’e-service. 
```

**Fase 3: Implementazione**

> **Obiettivo**: adottare i requisiti che servono a realizzare le interfacce tecnologiche (API) necessarie per dialogare con i Credential Issuer gestendo l'intero ciclo di vita del dato. 

Reviewer: ```suggestion
Implementare le interfacce tecnologiche necessarie per comunicare con il Fornitore di Attestati Elettronici e gestire l’intero ciclo di vita dei dati associati ai (Q)EAA. 
```

> - **Sezione** :ref:`endpoints:Endpoints`: per implementare correttamente le interfacce (API) che permettono al Credential Issuer di recuperare i dati autorevoli in modo sicuro e standardizzato (riferimento a paragrafo :ref:`authentic-source-endpoint:Endpoint delle Fonti Autentiche`) 

Reviewer: ```suggestion
- **Sezione** :ref:`authentic-sources:Fonti Autentiche`: Ruolo e responsabilità della Fonte Autentica. 
```

> 

Reviewer: ```suggestion
**Sezione** :ref:`e-service-pdnd:e-Service PDND`: Specifiche obbligatorie per l’integrazione con la PDND (Piattaforma Digitale Nazionale Dati) e relativi requisiti di interoperabilità per la pubblicazione di un e-service.

**Sezione** :ref:`authentic-source-endpoint:Endpoint delle Fonti Autentiche`: Requisiti chiave per l’implementazione delle API che consentono al Credential Issuer di recuperare in modo sicuro e coerente i dati autorevoli e di gestirne il ciclo di vita tramite gli endpoint del Signal Hub. 
```

> - **Sezione** :ref:`digital-credential-management:Gestione degli Attestati Elettronici`: per conoscere nello specifico il ciclo di vita e la gestione degli Attestati elettronici (riferimento al paragrafo :ref:`credential-revocation:Ciclo di vita degli Attestati Elettronici`) 

Reviewer: ```suggestion

```

> 

Reviewer: ```suggestion
- **Sezione** :ref:`registry infrastructure:Infrastruttura del Registro`: Approfondimento sui componenti del Registro di interesse per la Fonte Autentica. 
```

> - **Sezione** :ref:`security-privacy-considerations:Considerazioni di Sicurezza e Privacy` per implementare soluzioni che rispettino tutti i parametri di sicurezza  

Reviewer: ```suggestion

```

> - **Sezione** :ref:`log-retention-policy:Politiche Generali di Conservazione dei Log`: per implementare soluzioni che rispettino i parametri di sicurezza e i requisiti definiti nella norma ISO/IEC 27001 

Reviewer: ```suggestion
- **Sezione** :ref:`log-retention-policy:Politiche Generali di Conservazione dei Log`: Requisiti generali per la conservazione dei log e requisiti specifici per le Fonti Autentiche, in conformità alla norma ISO/IEC 27001. 
```

> 

Reviewer: ```suggestion
- **Sezione** :ref:`test-plans:Test Plans`: Guida alla configurazione dell’ambiente di test e alla validazione delle interazioni backend tramite l’utilizzo delle matrici di test fornite dall’ecosistema. 
```

**Fase 4: Registrazione**

> **Obiettivo**: accreditarsi come Fonte Autentica nel sistema rispettando le procedure amministrative e tecniche che garantiscono sull’affidabilità dell'ente. 

Reviewer: ```suggestion
Registrarsi al sistema come Fonte Autentica, completando le procedure amministrative e tecniche richieste. 
```

> - **Sezione** :ref:`onboarding-high-level:Sistema di Onboarding`: per conoscere le modalità di partecipazione all’ecosistema IT-Wallet e tutto ciò che riguarda la riconoscibilità dei propri dati da parte di tutti gli attori (riferimento al paragrafo :ref:`onboarding-high-level:Processo di registrazione della Fonte Autentica`).  

Reviewer: ```suggestion
- **Sezione** :ref:`onboarding-high-level:Sistema di Onboarding`: Panoramica dell’architettura del sistema di onboarding e del processo di registrazione della Fonte Autentica. 

- **Sezione** :ref:`entity-onboarding:Onboarding delle Entità`: Approfondimento sulle procedure tecniche per la registrazione della Fonte Autentica. 
```

> - **Sezione** :ref:`x5c-evaluation:Operazioni di Gestione dei Certificati X.509`: per conoscere le procedure operative per la gestione dei Certificati X.509 all'interno della federazione IT-Wallet. 

Reviewer: ```suggestion
- **Sezione** :ref:`x5c-evaluation:Operazioni di Gestione dei Certificati X.509`: Procedure operative per la gestione dei certificati X.509 nell’ambito della federazione IT-Wallet. 
```

> **Wallet Provider**

Reviewer: ```suggestion
**Fornitore di Wallet**
```

~~~~~~~~~~~~~~~~~~~

Il Wallet Provider è colui che progetta il portafoglio digitale, pubblico o privato. Funge da punto di contatto diretto con il cittadino e la sua sfida consiste nel coniugare un'esperienza utente fluida e intuitiva con i più alti standard di sicurezza. 

> **Focus**: rispetto dei protocolli e degli standard di sicurezza, fluidità dell’esperienza utente, rispetto delle linee guida del sistema IT-Wallet. 

Reviewer: ```suggestion
Il Fornitore di Wallet si occupa della progettazione e dello sviluppo della Soluzione Wallet, che consente all’Utente di conservare, gestire e presentare il proprio PID e i propri (Q)EAA. 
```

> **Fase 1: Scoperta del sistema IT-Wallet**

Reviewer: ```suggestion
**Fase 1: Scoperta**
```

> **Obiettivo:** conoscere il contesto di riferimento, i requisiti tecnici e la normativa per entrare a far parte della Trust Chain (catena di fiducia) su cui si fonda tutto il sistema. 

Reviewer: ```suggestion
Comprendere il funzionamento generale dell’ecosistema, l’architettura tecnica e l’Infrastruttura di Trust. 
```

- **Sezione** :ref:`introduction:Introduzione`: per comprendere l’ambito e il linguaggio normativo dell'ecosistema IT-Wallet. 

- **Sezione** :ref:`architecture-overview:Panoramica dell'Architettura`: per conoscere l’Architettura del sistema IT-Wallet, in termini di governance e processi operativi abilitati.  

- **Sezione** :ref:`trust-infrastructure:L'Infrastruttura di Trust`: per comprendere il funzionamento della Trust Chain e capire come validare gli attestati delle Fonti Autentiche (AS) e degli Issuer (CI). 

> - **Sezione** :ref:`defined-terms-and-references:Termini Definiti e Riferimenti`: per conoscere i riferimenti normativi, i termini definiti e gli standard tecnici per consentire un'interoperabilità sicura e corretta tra tutti i partecipanti (puoi ricorrere a questa sezione per ogni dubbio su terminologia, standard di riferimento, acronimi...). 

Reviewer: ```suggestion
- **Sezione** :ref:`introduction:Introduzione`: Scopo e contesto normativo del Sistema IT-Wallet.

- **Sezione** :ref:`architecture-overview:Panoramica dell'Architettura`: Panoramica dell’architettura del Sistema IT-Wallet in termini di governance e processi operativi abilitati. 

- **Sezione** :ref:`trust-infrastructure:L'Infrastruttura di Trust`: Requisiti chiave del modello di fiducia federato e dei meccanismi di valutazione della fiducia tra le entità.

- **Sezione** :ref:`defined-terms-and-references:Termini Definiti e Riferimenti`: Terminologia completa, riferimenti normativi, documentazione, risorse e strumenti aggiuntivi, linee guida per i contributi. 
```

**Fase 2: Design**

> **Obiettivo**: definire l'identità visiva del Wallet, progettare l’esperienza utente, modellare l'entità tecnica e scegliere gli algoritmi crittografici che garantiranno la protezione dei dati sul dispositivo. 

Reviewer: ```suggestion
**Fase 2: Progettazione**

Comprendere i requisiti relativi all’Esperienza Utente e progettare la Soluzione Wallet secondo modelli comuni, al fine di garantirne l’usabilità e l’accessibilità. 
```

- **Sezione** :ref:`brand-identity:Brand Identity`: per garantire che l'App rispetti le linee guida di comunicazione e l'identità visiva del sistema IT-Wallet.  

- **Sezione** :ref:`functionalities:Design dell'Esperienza Utente`: per progettare interfacce semplici e sicure garantendo che l’esperienza utente sia immediata e intuitiva. 

> - **Sezione** :ref:`entities:Entità`: per conoscere i requisiti di implementazione e gli attributi necessari per configurare la propria entità (riferimento al paragrafo :ref:`Soluzione Wallet`). 

Reviewer: commento superato, da chiudere

**Sezione** :ref:`digital-credential-management:Gestione degli Attestati Elettronici`: per progettare come l'App gestirà tecnicamente il ciclo di vita dei dati (archiviazione protetta e presentazione). 

> **Sezione** :ref:`algorithms:Algoritmi Crittografici`: per selezionare e progettare l'implementazione degli standard crittografici necessari alla messa in sicurezza delle chiavi e delle transazioni. 

Reviewer: ```suggestion
- **Sezione** :ref:`brand-identity:Brand Identity`: Panoramica della Brand Identity del Sistema IT-Wallet e indicazioni sugli asset di riferimento per il Fornitore di Wallet.

- **Sezione** :ref:`functionalities:Design dell'Esperienza Utente`: Requisiti chiave relativi ai modelli di interazione, ai layout delle interfacce e agli asset grafici, al fine di garantire un’esperienza utente efficace, fluida e assicurare coerenza tra le diverse Soluzioni Wallet.

**Sezione** :ref:`digital-credential-management:Gestione degli Attestati Elettronici`: Gestione degli Attestati Elettronici: Requisiti tecnici e funzionali relativi al ciclo di vita dei (Q)EAA. 
```

**Fase 3: Implementazione**

> **Obiettivo**: implementare gli endpoint per la comunicazione e adottare gli standard tecnologici (es. SD-JWT) che assicurano al Wallet di interfacciarsi con gli altri attori.

Reviewer: ```suggestion
Implementare la Soluzione Wallet in conformità a specifici standard tecnologici, al fine di garantire la comunicazione tra la Soluzione Wallet e gli altri attori del Sistema. 
```

> - **Sezione** ref:`endpoints:Endpoints`: per implementare correttamente le interfacce (API) del Wallet Provider necessarie all'interoperabilità del sistema (riferimento al paragrafo ref:`endpoints:Endpoint del Fornitore di Wallet`). 

Reviewer: commento superato, da chiudere

- **Sezione** :ref:`security-privacy-considerations:Considerazioni di Sicurezza e Privacy`: per implementare soluzioni che rispettino tutti i parametri di sicurezza.  

> - **Sezione** :ref:`log-retention-policy:Politiche Generali di Conservazione dei Log`: per implementare soluzioni che rispettino i parametri di sicurezza e i requisiti definiti nella norma ISO/IEC 27001.

Reviewer: ```suggestion
- **Sezione** :ref:`wallet-solution:Soluzione Wallet`: Requisiti tecnici e funzionali relativi ai componenti, alle funzionalità e al ciclo di vita necessari per la configurazione della Soluzione Wallet. 

- **Sezione** :ref:`digital-credential-flows: Flussi relativi agli Attestati Elettronici`: Requisiti tecnici e funzionali relativi ai flussi di emissione e presentazione dei (Q)EAA.

- **Sezione** :ref:`wallet-provider-endpoint:Endpoint del Fornitore di Wallet`: Requisiti chiave per l’implementazione delle interfacce (API) del Fornitore di Wallet necessarie a garantire l’interoperabilità tra le entità. 

- **Sezione** :ref:`registry-infrastructure:Infrastruttura del Registro`: Approfondimento sui componenti del Registro. 

- **Sezione** :ref:`algorithms:Algoritmi Crittografici’: Selezione e implementazione degli standard crittografici necessari a garantire la sicurezza delle chiavi e delle transazioni. 

- **Sezione** :ref:`security-privacy-considerations:Considerazioni di Sicurezza e Privacy`: Requisiti di sicurezza e conformità per le Soluzioni Wallet. 

- **Sezione** :ref:`log-retention-policy:Politiche Generali di Conservazione dei Log`: Requisiti generali per la conservazione dei log e requisiti specifici per i Fornitori di Wallet, in conformità alla norma ISO/IEC 27001. 

- **Sezione** :ref:`mobile-application-instance: Istanza dell'Applicazione Mobile`: Requisiti relativi all’istanza dell’applicazione mobile, con riferimento alla richiesta e alla risposta di inizializzazione. 

- **Sezione** :ref:`test-plans:Test Plans`: Guida alla configurazione dell’ambiente di test e alla validazione delle interazioni backend tramite l’utilizzo delle matrici di test fornite dall’ecosistema. 
```

**Fase 4: Registrazione**

> **Obiettivo**: accreditarsi come Wallet Provider nel sistema in modo tale che le Wallet Instance dei cittadini siano riconosciute come valide dal sistema.  

Reviewer: ```suggestion
Registrarsi al sistema come Fornitore di Wallet, completando le procedure amministrative e tecniche richieste per il riconoscimento della Soluzione Wallet da parte del sistema. 
```

- **Sezione** :ref:`onboarding-high-level:Sistema di Onboarding`: per conoscere le modalità di partecipazione all’ecosistema IT-Wallet e tutto ciò che riguarda la riconoscibilità dei propri dati da parte di tutti gli attori (riferimento al paragrafo :ref:`onboarding-high-level:Journey dell'Operatore del Fornitore di Wallet`).  

- **Sezione** :ref:`x5c-evaluation:Operazioni di Gestione dei Certificati X.509`: per conoscere le procedure operative per la gestione dei Certificati X.509 all'interno della federazione IT-Wallet. 

> - **Sezione** :ref:`test-plans:Test Plans`: per avere informazioni in merito a come gli ambienti di test possono supportare implementatori, auditor e ambienti di test di conformità nella validazione del comportamento delle Soluzioni Wallet.

Reviewer: ```suggestion
- **Sezione** :ref:`onboarding-high-level:Sistema di Onboarding`: Panoramica dell’architettura del sistema di onboarding e del processo di registrazione del Fornitore di Wallet.

- **Sezione** :ref:`entity-onboarding:Onboarding delle Entità`: Approfondimento sulle procedure tecniche per la registrazione del Fornitore di Wallet. 

- **Sezione** :ref:`x5c-evaluation:Operazioni di Gestione dei Certificati X.509`: Procedure operative per la gestione dei certificati X.509 nell’ambito della federazione IT-Wallet. 

```

Fornitori di Credenziali (Credential Issuer)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Il Credential Issuer è il fornitore tecnico delle credenziali. Il suo compito principale è trasformare i dati ricevuti dalle Fonti Autentiche in attestati digitali sicuri, pronti per essere emessi verso il Wallet. È il fulcro tecnico che garantisce l'interoperabilità attraverso l'uso di standard internazionali come OpenID4VCI. 

> **Focus**: progettazione delle credenziali a partire dal dato grezzo, sicurezza del dato ottenuto, ciclo di vita delle credenziali. 

Reviewer: ```suggestion
Fornitore di Attestati Elettronici 
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Il Fornitore di Attestati Elettronici si occupa di trasformare i dati autorevoli di origine provenienti dalle Fonti Autentiche in (Q)EAA e di gestirne l’intero ciclo di vita, dall’emissione alla revoca o alla scadenza. 
```

**Fase 1: Scoperta del sistema IT-Wallet**

**Obiettivo**: conoscere il contesto di riferimento, i requisiti tecnici e la normativa per entrare a far parte della Trust Chain (catena di fiducia) su cui si fonda tutto il sistema. 

- **Sezione** :ref:`introduction:Introduzione`: per comprendere l’ambito e il linguaggio normativo dell'ecosistema IT-Wallet. 

- **Sezione** :ref:`architecture-overview:Panoramica dell'Architettura`: per conoscere l’Architettura del sistema IT-Wallet, in termini di governance e processi operativi abilitati.  

- **Sezione** :ref:`trust-infrastructure:L'Infrastruttura di Trust`: per comprendere come il Credential Issuer viene riconosciuto nel sistema del Wallet come emittente autorizzato e come deve validare a sua volta le Fonti Autentiche.

> - **Sezione** :ref:`defined-terms-and-references:Termini Definiti e Riferimenti`: per conoscere i riferimenti normativi, i termini definiti e gli standard tecnici per consentire un'interoperabilità sicura e corretta tra tutti i partecipanti (puoi ricorrere a questa sezione per ogni dubbio su terminologia, standard di riferimento, acronimi...).

Reviewer: ```suggestion
**Fase 1: Scoperta**

Comprendere il funzionamento generale dell’ecosistema, l’architettura tecnica e l’Infrastruttura di Trust. 

- **Sezione** :ref:`introduction:Introduzione`: Scopo e contesto normativo del Sistema IT-Wallet.

- **Sezione** :ref:`architecture-overview:Panoramica dell'Architettura`: Panoramica dell’architettura del Sistema IT-Wallet in termini di governance e processi operativi abilitati. 

- **Sezione** :ref:`trust-infrastructure:L'Infrastruttura di Trust`: Requisiti chiave del modello di fiducia federato e dei meccanismi di valutazione della fiducia tra le entità.

- **Sezione** :ref:`defined-terms-and-references:Termini Definiti e Riferimenti`: Terminologia completa, riferimenti normativi, documentazione, risorse e strumenti aggiuntivi, linee guida per i contributi. 
```

**Fase 2: Design**

**Obiettivo**: progettare tecnicamente gli attestati strutturando i metadati affinché rispondano ai requisiti tecnici e normativi e permettano all'utente di condividere solo le informazioni necessarie. 

- **Sezione** :ref:`functionalities:Design dell'Esperienza Utente`: per conoscere i requisiti funzionali di alto livello a supporto dell’Esperienza Utente in tutte le fasi di interazione tra l’Utente e il servizio. 

> - **Sezione** :ref:`entities:Entità`: per conoscere i requisiti di implementazione e gli attributi necessari per configurare il proprio profilo di Issuer (riferimento ai paragrafi :ref:`Soluzione del Fornitore di Attestati Elettronici` e :ref:`aFonti Autentiche`).

Reviewer: commento obsoleto, da chiudere

**Sezione** :ref:`digital-credential-management:Gestione degli Attestati Elettronici`:  per conoscere i requisiti tecnici necessari per la progettazione e gestione degli attestati e il loro ciclo di vita. 

> **Sezione** :ref:`algorithms:Algoritmi Crittografici`: per rispondere ai requisiti di sicurezza progettando sistemi di firma digitale che rendano gli attestati non falsificabili.

Reviewer: ```suggestion
**Fase 2: Progettazione**

Comprendere i requisiti e progettare tecnicamente i (Q)EAA, strutturandone i metadati in conformità ai requisiti tecnici e normativi. 

- **Sezione** :ref:`functionalities:Design dell'Esperienza Utente`: Requisiti chiave relativi alla struttura, all’emissione, allo stato e alla gestione nel tempo dei (Q)EAA.

**Sezione** :ref:`digital-credential-management:Gestione degli Attestati Elettronici`:  Requisiti tecnici e funzionali relativi al ciclo di vita dei (Q)EAA. 
```

**Fase 3: Implementazione**

**Obiettivo**: sviluppare gli endpoint di emissione in base al protocollo OpenID4VCI, implementando le funzioni di rilascio, rinnovo e gestione tecnica del ciclo di vita della credenziale.

- **Sezione** ref:`digital-credential-flows:Flussi relativi agli attestati elettronici`: per conoscere le specifiche di dettaglio per implementare i flussi di emissione, le modalità di presentazione dell’attestato elettronico e il flusso di recupero delle informazioni del wallet. 

- **Sezione** ref:`endpoints:Endpoints`: per implementare correttamente le interfacce (API) di emissione, in particolare il Credential Endpoint e il Token Endpoint  (riferimento al paragrafo ref:`endpoints:Endpoint del Credential Issuer`). 

- **Sezione** :ref:`security-privacy-considerations:Considerazioni di Sicurezza e Privacy`: per implementare soluzioni che rispettino tutti i parametri di sicurezza.  

> - **Sezione** :ref:`log-retention-policy:Politiche Generali di Conservazione dei Log`: per implementare soluzioni che rispettino i parametri di sicurezza e i requisiti definiti nella norma ISO/IEC 27001.

Reviewer: ```suggestion
**Fase 3: Implementazione**

Sviluppare endpoint secondo su specifici protocolli e implementare le funzionalità di emissione, rinnovo, revoca e tutte le altre funzioni tecniche di gestione del ciclo di vita dei (Q)EAA. 

- **Sezione** :ref:`credential-issuer-solution:Soluzione del Fornitore di Attestati Elettronici`: Requisiti tecnici e funzionali relativi ai componenti e ai modelli di interazione necessari per l’emissione e la gestione del ciclo di vita dei (Q)EAA. 

- **Sezione** :ref:`digital-credential-flows:Flussi relativi agli attestati elettronici`: Requisiti tecnici e funzionali relativi ai flussi di emissione e presentazione dei (Q)EAA.

- **Sezione** :ref:`credential-issuer-endopoint:Endpoint del Credential Issuer`: Requisiti chiave per l’implementazione dei metadati del Fornitore di Attestati Elettronici e degli endpoint di autorizzazione. 

- **Sezione** :ref:`algorithms:Algoritmi Crittografici’: Selezione e implementazione degli standard crittografici necessari a garantire la sicurezza delle chiavi e delle transazioni. 

- **Sezione** :ref:`e-service-pdnd:e-Service PDND’: Specifiche obbligatorie per l’integrazione con la PDND (Piattaforma Digitale Nazionale Dati) e relativi requisiti di interoperabilità per la pubblicazione di un e-service. 

- **Sezione** :ref:`security-privacy-considerations:Considerazioni di Sicurezza e Privacy`: Requisiti di sicurezza e conformità per le soluzioni implementate. 

- **Sezione** :ref:`log-retention-policy:Politiche Generali di Conservazione dei Log`: Requisiti generali per la conservazione dei log e requisiti specifici per i Fornitori di Attestati Elettronici, in conformità alla norma ISO/IEC 27001. 

- **Sezione** :ref:`test-plans:Test Plans`: Guida alla configurazione dell’ambiente di test e alla validazione delle interazioni backend tramite l’utilizzo delle matrici di test fornite dall’ecosistema. 
```

**Fase 4: Registrazione**

**Obiettivo**: accreditarsi nel sistema in modo tale che gli attestati emessi verso il Wallet risultino ufficialmente “fidati” e verificabili. 

- **Sezione** :ref:`onboarding-high-level:Sistema di Onboarding`: per conoscere le modalità di partecipazione all’ecosistema IT-Wallet  (riferimento al paragrafo :ref:`onboarding-high-level:Journey dell'Operatore del Credential Issuer`).  

- **Sezione** :ref:`x5c-evaluation:Operazioni di Gestione dei Certificati X.509`: per conoscere le procedure operative per la gestione dei Certificati X.509 all'interno della federazione IT-Wallet. 

> - **Sezione** :ref:`test-plans:Test Plans`: per avere informazioni in merito a come gli ambienti di test possono supportare implementatori, auditor e ambienti di test di conformità nella validazione del comportamento delle Soluzioni Wallet.

Reviewer: ```suggestion
**Fase 4: Registrazione**

Registrarsi al sistema come Credential Issuer, completando le procedure amministrative e tecniche previste affinché i (Q)EAA emessi verso i Wallet siano riconosciuti come affidabili dal Sistema. 

- **Sezione** :ref:`onboarding-high-level:Sistema di Onboarding`: Panoramica dell’architettura del sistema di onboarding e del processo di registrazione della Fonte Autentica. 

- **Sezione** :ref:`Entity Onboarding:Onboarding delle Entità`: Approfondimento sulle procedure tecniche per la registrazione del Fornitore di Attestati Elettronici. 

- **Sezione** :ref:`x5c-evaluation:Operazioni di Gestione dei Certificati X.509`: Procedure operative per la gestione dei certificati X.509 nell’ambito della federazione IT-Wallet. 
```

Fornitori di servizi (Relying Party)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Il Relying Party è l'utilizzatore finale delle credenziali. Il suo compito è trasformare la verifica digitale in un valore aggiunto per il proprio business, garantendo al contempo all'utente un processo di identificazione rapido, sicuro e rispettoso della minimizzazione dei dati richiesti. 

> **Focus**: progettazione dell’esperienza di verifica delle credenziali, rispetto della privacy dell’utente  

Reviewer: ```suggestion
Relying Party
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

La Relying Party si occupa di richiedere, ricevere e verificare in modo sicuro l’autenticità e la validità del PID e dei (Q)EAA presentati dall’Utente, al fine di consentire l’accesso a servizi online e offline. 
```

**Fase 1: Scoperta del sistema IT-Wallet**

**Obiettivo**: conoscere il contesto di riferimento, i requisiti tecnici e la normativa per entrare a far parte della Trust Chain (catena di fiducia) su cui si fonda tutto il sistema.

- **Sezione** :ref:`introduction:Introduzione`: per comprendere l’ambito e il linguaggio normativo dell'ecosistema IT-Wallet. 

- **Sezione** :ref:`architecture-overview:Panoramica dell'Architettura`: per conoscere l’Architettura del sistema IT-Wallet, in termini di governance e processi operativi abilitati.   

- **Sezione** :ref:`trust-infrastructure:L'Infrastruttura di Trust`: per capire come verificare che la credenziale presentata dall'utente provenga da una Fonte Autentica certificata.

> - **Sezione** :ref:`defined-terms-and-references:Termini Definiti e Riferimenti`: per conoscere i riferimenti normativi, i termini definiti e gli standard tecnici per consentire un'interoperabilità sicura e corretta tra tutti i partecipanti (puoi ricorrere a questa sezione per ogni dubbio su terminologia, standard di riferimento, acronimi...).

Reviewer: ```suggestion
**Fase 1: Scoperta*

Comprendere il funzionamento generale dell’ecosistema, l’architettura tecnica e l’Infrastruttura di Trust. 

- **Sezione** :ref:`introduction:Introduzione`: Scopo e contesto normativo del Sistema IT-Wallet.

- **Sezione** :ref:`architecture-overview:Panoramica dell'Architettura`: Panoramica dell’architettura del Sistema IT-Wallet in termini di governance e processi operativi abilitati. 

- **Sezione** :ref:`trust-infrastructure:L'Infrastruttura di Trust`: Requisiti chiave del modello di fiducia federato e dei meccanismi di valutazione della fiducia tra le entità.

- **Sezione** :ref:`defined-terms-and-references:Termini Definiti e Riferimenti`: Terminologia completa, riferimenti normativi, documentazione, risorse e strumenti aggiuntivi, linee guida per i contributi. 
```

**Fase 2: Design**

> **Obiettivo**: definire i requisiti di verifica, stabilendo gli attributi necessari per erogare il servizio; progettare l'esperienza di richiesta, definendo le modalità di condivisione dei dati. 

Reviewer: ```suggestion
**Fase 2: Progettazione**

Comprendere i requisiti di Esperienza Utente e progettare le funzionalità di verifica necessarie all’erogazione del servizio all’Utente finale. 
```

> - **Sezione** :ref:`brand-identity:Brand Identity`: per garantire che le eventuali app di verifica rispettino le linee guida di comunicazione e l'identità visiva del sistema IT-Wallet.  

Reviewer: ```suggestion
- **Sezione** :ref:`brand-identity:Brand Identity`: Panoramica della Brand Identity del Sistema IT-Wallet e indicazioni sugli asset da adottare da parte della Relying Party.
```

> - **Sezione** :ref:`functionalities:Design dell'Esperienza Utente`: per conoscere i requisiti funzionali di alto livello a supporto dell’Esperienza Utente in tutte le fasi di interazione tra l’Utente e il servizio.  

Reviewer: ```suggestion
- **Sezione** :ref:`functionalities:Design dell'Esperienza Utente`: Requisiti chiave relativi ai modelli di interazione, ai layout delle interfacce e agli asset grafici, al fine di garantire un’esperienza utente efficace, fluida e assicurare coerenza tra i sistemi di presentazione e verifica. 
```

> - **Sezione** :ref:`entities:Entità`: per conoscere i requisiti di implementazione e gli attributi necessari per configurare il profilo del Verificatore (riferimento al paragrafo :ref:`Soluzione di Relying Party`). 

Reviewer: ```suggestion

```

> **Sezione** :ref:`digital-credential-management:Gestione degli Attestati Elettronici`: per comprendere quali sono i formati delle Credenziali Elettroniche e verifica della validità. 

Reviewer: ```suggestion
**Sezione** :ref:`digital-credential-management:Gestione degli Attestati Elettronici`: Gestione degli Attestati Elettronici: Requisiti tecnici e funzionali relativi al ciclo di vita dei (Q)EAA.
```

> **Sezione** :ref:`algorithms:Algoritmi Crittografici`: per progettare i sistemi di validazione delle firme digitali che "sigillano" i dati presentati dall'utente. 

Reviewer: ```suggestion

```

**Fase 3: Implementazione**

> **Obiettivo**: implementare il protocollo OpenID4VP per inviare le richieste al Wallet e ricevere gli attestati, assicurando la corretta decodifica dei formati. 

Reviewer: ```suggestion
Implementare le funzionalità di verifica secondo specifici protocolli, al fine di inviare richieste di verifica al Wallet e ricevere una risposta con l’autorizzazione dell’Utente. 
```

> - **Sezione** ref:`digital-credential-flows:Flussi relativi agli attestati elettronici`: per l’implementazione del flusso di presentazione sia per scenari remoti che di prossimità (ref:`credential-presentation:12.2`. 

Reviewer: ```suggestion

```

> - **Sezione** ref:`endpoints:Endpoints`: per implementare correttamente le interfacce di ricezione e verifica dei dati (riferimento ai paragrafi ref:`relying-party-endpoints:Endpoint della Relying Party` e ref:`relying-party-endpoints:Endpoint del Backend del Provider di Relying Party`). 

Reviewer 2: ```suggestion
- **Sezione** :ref:`endpoints:Endpoints`: per implementare correttamente le interfacce di ricezione e verifica dei dati (riferimento ai paragrafi :ref:`relying-party-endpoints:Endpoint della Relying Party` e :ref:`relying-party-endpoints:Endpoint del Backend del Provider di Relying Party`). 
```

> - **Sezione** :ref:`endpoints:Endpoints`: per implementare correttamente le interfacce di ricezione e verifica dei dati (riferimento ai paragrafi :ref:`relying-party-endpoints:Endpoint della Relying Party` e :ref:`relying-party-endpoints:Endpoint del Backend del Provider di Relying Party`). 

Reviewer: ```suggestion
- **Sezione** :ref:`relying-party-solution:Soluzione di Relying Party`: Requisiti tecnici e funzionali relativi ai componenti e alle funzionalità per la verifica del PID e dei (Q)EAA. 

- **Sezione** :ref:`digital-credential-flows:Flussi relativi agli Attestati Elettronici`: Requisiti tecnici e funzionali relativi ai flussi di emissione e presentazione dei (Q)EAA. 

- **Sezione** :ref:`relying-party-endpoint:Endpoint della Relying Party`: Requisiti chiave per l’implementazione degli endpoint di verifica dei (Q)EAA.
```

> - **Sezione** :ref:`security-privacy-considerations:Considerazioni di Sicurezza e Privacy`: per implementare soluzioni che rispettino tutti i parametri di sicurezza.  

Reviewer: ```suggestion
- **Sezione** :ref:`security-privacy-considerations:Considerazioni di Sicurezza e Privacy`: Requisiti di sicurezza e conformità per le soluzioni implementate. 
```

> - **Sezione** :ref:`log-retention-policy:Politiche Generali di Conservazione dei Log`: per implementare soluzioni che rispettino i parametri di sicurezza e i requisiti definiti nella norma ISO/IEC 27001.

Reviewer: ```suggestion
- **Sezione** :ref:`log-retention-policy:Politiche Generali di Conservazione dei Log`: Requisiti generali per la conservazione dei log e requisiti specifici per le Relying Party, in conformità alla norma ISO/IEC 27001. 

- **Sezione** :ref:`test-plans:Test Plans`: Guida alla configurazione dell’ambiente di test e alla validazione delle interazioni backend tramite l’utilizzo delle matrici di test fornite dall’ecosistema. 
```

**Fase 4: Registrazione**

> **Obiettivo**: accreditarsi nel sistema in modo tale da essere riconosciuti come attore sicuro e affidabile nel momento della richiesta dei dati dell’utente.  

Reviewer: ```suggestion
Registrarsi al sistema come Relying Party, completando le procedure amministrative e tecniche previste, e diventare soggetto affidabile per la richiesta dei dati degli Utenti. 
```

> - **Sezione** :ref:`onboarding-high-level:Sistema di Onboarding`: per conoscere le modalità di partecipazione all’ecosistema IT-Wallet (riferimento al paragrafo :ref:`onboarding-high-level:Journey dell'Operatore della Relying Party`).  

Reviewer: ```suggestion
- **Sezione** :ref:`onboarding-high-level:Sistema di Onboarding`: Approfondimento sulle procedure tecniche per la registrazione della Relying Party. 

- **Sezione** :ref:`entity Onboarding:Onboarding delle Entità`: Approfondimento sulle procedure tecniche per la registrazione della Relying Party. 
```

> - **Sezione** :ref:`x5c-evaluation:Operazioni di Gestione dei Certificati X.509`: per conoscere le procedure operative per la gestione dei Certificati X.509 all'interno della federazione IT-Wallet. 

Reviewer: ```suggestion
- **Sezione** :ref:`x5c-evaluation:Operazioni di Gestione dei Certificati X.509`: Procedure operative per la gestione dei certificati X.509 nell’ambito della federazione IT-Wallet. 
```

> - **Sezione** :ref:`test-plans:Test Plans`: per avere informazioni in merito a come gli ambienti di test possono supportare implementatori, auditor e ambienti di test di conformità nella validazione del comportamento delle Soluzioni Wallet.

Reviewer: ```suggestion

```
